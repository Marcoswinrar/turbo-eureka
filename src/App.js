import './App.css';
import ListExpand from './components/list-expand';

function App() {
  return (
    <div className="App">
      <h2>React API</h2>
      <ListExpand />
    </div>
  )
}

export default App
