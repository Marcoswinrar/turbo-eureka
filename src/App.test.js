import { render, screen } from "@testing-library/react"
import App from './App'

describe("Render do <App/>", () => {
  
  it("Deve renderizar o App sem problemas", () => {
    const { container } = render(<App />)

    expect(container).toMatchSnapshot()
    expect(screen.queryByText(/React API/)).toBeInTheDocument()
  })
})