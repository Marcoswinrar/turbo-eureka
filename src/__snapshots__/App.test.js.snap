// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <div
    class="App"
  >
    <h2>
      React API
    </h2>
    <div
      class="sc-hKMtZM HhcTB"
    >
      <section
        class="sc-eCYdqJ kYIXbH"
      >
        <header
          class="sc-gKXOVf jwCYRe"
        >
          <h3
            class="sc-jSMfEi bWEXfk"
          >
            Hooks - useEffect
          </h3>
        </header>
      </section>
    </div>
  </div>
</div>
`;

exports[`Render do <App/> Deve renderizar o App sem problemas 1`] = `
<div>
  <div
    class="App"
  >
    <h2>
      React API
    </h2>
    <div
      class="sc-hKMtZM HhcTB"
    >
      <section
        class="sc-eCYdqJ kYIXbH"
      >
        <header
          class="sc-gKXOVf jwCYRe"
        >
          <h3
            class="sc-jSMfEi bWEXfk"
          >
            Hooks - useEffect
          </h3>
        </header>
      </section>
    </div>
  </div>
</div>
`;
